packages/wonder-stuff-server-google/src/setup-integrations.js

```diff
@@ -0,0 +1,31 @@
+// @flow
```

Comment:
Copied from render-gateway's `setup-stackdriver.js` but renamed since Stackdriver isn't a thing now, and defaulted values to `false` so these are opt-in.